client/my-sites/scan/wpcom-atomic-transfer.tsx

```diff
@@ -0,0 +1,26 @@
+import { translate } from 'i18n-calypso';
+import JetpackScanSVG from 'calypso/assets/images/illustrations/jetpack-scan.svg';
+import wpcomAtomicTransfer from 'calypso/lib/jetpack/wpcom-atomic-transfer';
+import WPCOMScanUpsellPage from 'calypso/my-sites/scan/wpcom-upsell';
+
+export function wpcomJetpackScanAtomicTransfer() {
+	const content = {
+		documentHeadTitle: translate( 'Activate Jetpack Scan now' ) as string,
+		header: translate( 'Jetpack Scan' ) as string,
```

Comment:
No change needed because I bet this is already a translated string....but do we translate product names?